calculation.js

```diff
@@ -0,0 +1,14 @@
+const strictEqual = (inputOne, inputTwo) => {
+  return types(inputOne, inputTwo) && values(inputOne, inputTwo)
+}
+
+const types = (inputOne, inputTwo) => {
+  return typeof inputOne === typeof inputTwo
+}
```

Comment:
comparison operator ( == ) in JavaScript is used for comparing two variables, but it ignores the datatype of variable. This would have also worked to avoid using ( ===) comparison operator